fetch.py

```diff
@@ -131,6 +217,12 @@ def validate_raw_checksum(conn: openstack.connection.Connection,
     """
     Compares the currently existing raw file's md5 checksum to the one on openstack.
     Returns true if they match
+
+    Returns
+    -------
+    bool
+        True if the MD5 hash of the local raw file is the same as the current one on openstack
+        False otherwise
     """
     cur_img = next(conn.image.images(
```

Comment:
here the assumption is that there will be a single image returned with those properties. We should perform a check similar to the one in function `validate_checksums`, i.e., log a warning if there are more than one images returned